scripts/ci.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
```

Comment:
Probably irrelevant in this case, but I tend to use `#!/usr/bin/env bash` instead (the gradle shellscript uses it too).